tests/functional/requirements.txt

```diff
@@ -1 +1,3 @@
 python-openstackclient
+# https://github.com/juju/python-libjuju/issues/1184
```

Comment:
This issue is now closed; can we remove the pin already? :)

Reply:
This is not backported to libjuju 2.x

Reply:
Ok, in that case, could you add that to the comment here to clarify? Thanks :)